terms/neuro/bodyWeight.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://repo-prod.prod.sagebase.org/repo/v1/schema/type/registered/sage.annotations-neuro.bodyWeight-0.0.1",
+    "type": "number",
+    "description": "Weight of animal in grams"
```

Comment:
Is there a way to make this key apply to humans too? I guess if we specify the units in it, it would be weird to have a human weight in grams.

Reply:
Nevermind, let's stick with this for now because the alternatives are worse.